controllers/emails/send.js

```diff
@@ -87,7 +87,13 @@ async function sendContactEmail(name, contactEmail, body) {
  */
 async function approvedEmail(recipientEmail) {
   subject = "Your submission was approved";
-  body = [];
+  body = [
+    "Your submission was finally reviewed! Hurray!",
+    " ",
+    "It has now need added to our collection of articles.",
```

Comment:
This should presumably be `It has now been added to our collection of articles.`